runtime/core/exec_aten/util/tensor_util.h

```diff
@@ -893,16 +894,6 @@ inline bool tensor_is_scalar(executorch::aten::Tensor t) {
   return t.dim() == 0 && t.numel() == 1;
 }
 
-/**
- * The expected output size may not be the existing size of any inputs and
- * outputs if the operator supports both broadcast and dynamic shape.
- * Therefore such operators needs extra space to store the calculated expected
- * output size. such dynamic allocation is troublesome in executorch so we can
- * just hard code a static value of a relatively small value because users
- * don't create high dimensional tensors.
- */
-constexpr size_t kTensorDimensionLimit = 16;
```

Comment:
Removing this is technically a BC-breaking change to the public API. To follow our policy we'd need to mark this as ET_DEPRECATED and keep it around before removing it in release 0.8.

Reply:
Oh ok I see it moved to a different header, and this header includes that one.